packages/components/src/Form/Inputs/InlineInputText/InlineInputText.story.tsx

```diff
@@ -69,3 +69,18 @@ ReadOnly.args = {
 ReadOnly.parameters = {
   storyshots: { disable: true },
 }
+
+export const OverflowHiddenInlineInputText = () => (
+  <div
+    style={{
+      border: '1px solid',
+      overflow: 'hidden',
+      width: '100px',
+    }}
+  >
+    <InlineInputText />
```

Comment:
I'd preload this will a long value to show the fix more clearly :)

```suggestion
    <InlineInputText value="Long example value that should require scrolling to reach" />
```